src/posts/PostClass.ts

```diff
@@ -282,8 +283,41 @@ export class PostClass {
       throw e;
     }
   }
+
+  static async likeComment(req: Request, res: Response){
+    try{
+        const postId = req.params.postId;
+        const userId = req.params.userId;
+        const likeObj = {
+            userID: req.params.userId,
+            like: req.body.like,
+        }
+        //Checks to see user liking the comment exists
+        const user = await PostClass.dbConnection()
+        .collection("users").findOne({_id: new ObjectID(userId)})
+        if(!user){
+            throw console.error("The specified user ID doesnt exist or is Invalid")
+        }
+        //Make sure comment exists
+        const post = await PostClass.dbConnection()
+        .collection(COLLECTION.POSTS).findOne({_id: new ObjectID(postId)})
```

Comment:
you are checking if the post exist here just like i had, this isnt correct. you should be getting the post, and checking if the comment in that post exists @ShawnAli97

Reply:
Thank you for the review, I'm gonna work on it now and implement the correct logic.